provisioning/setup.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+# Setup utility for Hare to configure Hare related settings, e.g. logrotate,
+# report unsupported features, etc.
+
+import argparse
+import asyncio
+import json
+import logging
+import shutil
+import subprocess
+
+from eos.utils.product_features import unsupported_features
+
+
+def get_data_from_provisioner_cli(method, output_format="json") -> str:
+    rc = 1
+    try:
+        process = subprocess.run(['provisioner', f"{method}", f"--out={output_format}"],
```

Comment:
[nit] Use single-quoted strings for the sake of consistency.

Reply:
Used doubles quotes as there is variable dereferencing. Let me try with single quotes.

Reply:
@mssawant Python makes no semantical distinction between single-quoted and double-quoted strings.
```
>>> s = 'world'
>>> f'Hello, {s}!'
'Hello, world!'
>>> "{}ds don't come easy".format(s[:3])
"words don't come easy"
```

Reply:
@mssawant `f'{method}'` should be replaced with `method`.

Reply:
ah you are right, no need to convert it to fstring, thanks.